src/components/Book.js

```diff
@@ -1,16 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 const Book = ({ book, handleRemoveBook }) => {
   const handleBookRemove = () => {
     handleRemoveBook(book);
   };
+  const percentage = 66;
   return (
-    <tr key={book.id} id={book.id}>
-      <td>{book.title}</td>
-      <td>{book.category}</td>
-      <td><button type="button" onClick={handleBookRemove} className="btn btn-danger">Delete</button></td>
-    </tr>
+    <div className="card">
+      <div className="row card-inside">
+        <div className="col-12 col-sm-3 col-md-4">
+          <p className="gender-book">{book.category}</p>
+          <p className="title-book">{book.title}</p>
+          <p className="author-book">JRR Tolkien</p>
+          <div className="row">
+            <div className="col-12 col-lg-5">
+              <a className="book-option" href="/">Comments</a>
+            </div>
+            <div className="col-12 col-lg-4">
+              <a className="book-option" href="/">Remove</a>
+            </div>
+            <div className="col-12 col-lg-3">
+              <a className="book-option" href="/">Edit</a>
+            </div>
+          </div>
+        </div>
+        <div className="col-6 col-sm-3 col-md-2 mt-4">
+          <CircularProgressbar value={percentage} />
+        </div>
+        <div className="col-12 col-sm-3 col-md-2">
+          <h1>28%</h1>
+          <p>Completed</p>
+        </div>
+        <div className="col-12 col-sm-3 col-md-4">
+          <p className="current-chapter">Current chapter</p>
+          <p>Chapter 13</p>
+          <button type="button" onClick={handleBookRemove} className="btn btn-update">Delete</button>
+        </div>
```

Comment:
- [ ] This button is a placeholder and should match the design as "Update Progress" even though it has not functionality. Kindly move the remove book functionality to the remove button.

Reply:
- According to the milestone, this part is just change styling, if I delete the "delete button" functionality I need to change a lot of things inside the code because linters will ask me why I have a function that I don't use. I'm gonna change the text on the button hoping will be enough but next CR.
![image](https://user-images.githubusercontent.com/20563252/125811390-60d8e821-bccb-43d4-8343-b16344a842d5.png)

Reply:
Hello @vichuge on this part I would suggest you look at my review below thanks.